Doc: Draft a v0.5.0 RN.

qianouyang opened the pull request:
Signed-off-by: qouyang <qian.ouyang@intel.com>

qianouyang committed:
Update VERSION

Commit:
Merge pull request #7 from qianouyang/v0.5.0_version_update

Update VERSION

Commit:
Update Jenkinsfile

Commit:
Update Jenkinsfile

qianouyang committed:
Add UT for errorcode.go

qianouyang committed:
Remove unnecessary log message.

ionutnechita committed:
Update readme.md - increase storage and memory size

Update readme.md - increase storage and memory size

Description:
 - Day-0 host need more storage for build and run services
 - Nodes need more storage and memory for run services in cluster

Commit:
update credentials

Commit:
update credentials

Commit:
Merge branch 'main' into error_code_ut

Commit:
Update Jenkinsfile

Commit:
Update Jenkinsfile

Commit:
Merge branch 'edge_conductor_v0.5.0' into verify-size-memory-storage-readme

Commit:
Merge branch 'main' into error_code_ut

qianouyang committed:
Update errorcode_test.go

qianouyang committed:
Merge pull request #9 from qianouyang/error_code_ut

fix: Add UT for errorcode.go

qianouyang committed:
Draft a v0.5.0 RN.
Signed-off-by: qouyang <qian.ouyang@intel.com>

hustlzp1981 commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+| 2  |`CRI-O`                       |  1.23.2 | No   | No   | Yes  |
+| 3  |`Edge Software Provisioner`   |  2.0.3  | No   | Yes  | Yes† |
+| 4  |`Grafana Dashboard`           |  8.3.6  | Yes  | Yes  | Yes  |
+| 5  |`Intel-GPU-Plugin`            |  0.23   | No   | Yes  | No   |
```
0.24

sunausti commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+|----| ------------|---------|---------|-----------|---------------|------------|
+| 1  |`KIND`               |  0.14.0    | 1.24.2  | -- | -- |-- |
+| 2  |`RKE`                 |  1.3.12    | 1.23.7-rancher1-1(default)  | -- | -- |-- |
+| 3  |`CAPI(core)`      |  1.23.2 | ---   | ---   | Cert-manager v1.9.0 |verified with metal3 v1.1.1|
```
```suggestion
| 3  |`CAPI(core)`      |  1.2.0 | ---   | ---   | Cert-manager v1.9.0 | -- |
```

sunausti commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+| 1  |`KIND`               |  0.14.0    | 1.24.2  | -- | -- |-- |
+| 2  |`RKE`                 |  1.3.12    | 1.23.7-rancher1-1(default)  | -- | -- |-- |
+| 3  |`CAPI(core)`      |  1.23.2 | ---   | ---   | Cert-manager v1.9.0 |verified with metal3 v1.1.1|
+| 4  |`CAPI-BYOH`    |  1.1.2 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|
```
```suggestion
| 4  |`CAPI-BYOH`    |  1.2.0 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|
```

SidneyAn replied:
should be
| 4 |CAPI-BYOH | 0.2.0 | 1.23.5 | Crio v1.23.3 Containerd v1.6.6 | -- |---|

sunausti commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+| 2  |`RKE`                 |  1.3.12    | 1.23.7-rancher1-1(default)  | -- | -- |-- |
+| 3  |`CAPI(core)`      |  1.23.2 | ---   | ---   | Cert-manager v1.9.0 |verified with metal3 v1.1.1|
+| 4  |`CAPI-BYOH`    |  1.1.2 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|
+| 5  |`CAPI-METAL3`|  1.1.2 | 1.24.1   |  Crio v1.24.1 Containerd v1.6.6 | -- |---|
```
```suggestion
| 5  |`CAPI-METAL3`|  1.2.0 | 1.24.1   |  Crio v1.24.1 Containerd v1.6.6 | -- |---|
```

SidneyAn replied:
should be 
| 5  |`CAPI-METAL3`|  1.1.2  | 1.23.5   |  Crio v1.23.2 Containerd v1.6.6 | -- |ubuntu 22.04|

SidneyAn commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+| No.| EC v0.5.0 Features            | Version | Kind | RKE  | CAPI |
+|----| -----------------------------|---------|------|------|----- |
+| 1  |`Calico`                 |  3.23.1    | No  | No | Yes   |
+| 2  |`CRI-O`                       |  1.23.2 | No   | No   | Yes  |
```
1.23.3

SidneyAn commented on RELEASE-NOTES.md:
```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+| 1  |`KIND`               |  0.14.0    | 1.24.2  | -- | -- |-- |
+| 2  |`RKE`                 |  1.3.12    | 1.23.7-rancher1-1(default)  | -- | -- |-- |
+| 3  |`CAPI(core)`      |  1.23.2 | ---   | ---   | Cert-manager v1.9.0 |verified with metal3 v1.1.1|
+| 4  |`CAPI-BYOH`    |  1.1.2 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|
```
should be 
| 4  |`CAPI-BYOH`    |  0.2.0 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|

qianouyang committed:
Apply suggestions from code review

Co-authored-by: yhu6 <yong.hu@intel.com>

huix-shh committed:
test: node-join unit-test (#12)

* node-join unit-test

Signed-off-by: Huihuang Shi <huihuang.shi@intel.com>

qianouyang committed:
Apply suggestions from code review

Co-authored-by: sunausti <austin.sun@intel.com>

qianouyang committed:
Update RELEASE-NOTES.md

qianouyang committed:
Update RELEASE-NOTES.md

qianouyang committed:
Merge branch 'edge_conductor_v0.5.0' into cherrypick_ut_v0.5.0_20220908

Commit:
Merge pull request #18 from qianouyang/cherrypick_ut_v0.5.0_20220908

Cherrypick ut v0.5.0 20220908

ionutnechita committed:
Merge branch 'edge_conductor_v0.5.0' into verify-size-memory-storage-readme

qianouyang committed:
cherrypick: V0.5.0 cherry pick 20220914 (#23)

cherry pick below PR# from main to v0.5.0.
https://github.com/intel/edge-conductor/pull/16 (Update cert-manager image)
https://github.com/intel/edge-conductor/pull/19 (EPJ-3187)
https://github.com/intel/edge-conductor/pull/20 (Upgrade cert files)

qianouyang committed:
Apply suggestions from code review

Co-authored-by: jnadelbx <109433021+jnadelbx@users.noreply.github.com>

qianouyang committed:
Update RELEASE-NOTES.md

ionutnechita committed:
Merge branch 'edge_conductor_v0.5.0' into verify-size-memory-storage-readme

sunausti committed:
Merge pull request #10 from ionutnechita/verify-size-memory-storage-readme

doc: Update readme.md - increase storage and memory size
